packages/whosmysanta-backend/package.json

```diff
@@ -1,48 +1,21 @@
 {
   "name": "whosmysanta-backend",
   "version": "1.0.0",
-  "description": "GraphQL server for WhosMySanta",
-  "private": true,
-  "author": "WhosMySanta <hello@whosmysanta.com>",
+  "main": "index.js",
```

Comment:
Why do we want to do this? To keep us from accidentally publishing before we're ready to?